backend/src/process.py

```diff
@@ -177,9 +179,28 @@ async def __process(self, node: UsableData) -> Any:
             # Run the node and pass in inputs as args
             run_func = functools.partial(node_instance.run, *enforced_inputs)
             output = await self.loop.run_in_executor(None, run_func)
+            node_outputs = node_instance.get_outputs()
+            broadcast_data = dict()
```

Comment:
Please add a type. Then you'll see the bug from the next comment.
```suggestion
            broadcast_data: Dict[int, Any] = dict()
```